modules/nw-metallb-configure-address-pool.adoc

```diff
@@ -12,27 +12,26 @@ As a cluster administrator, you can add address pools to your cluster to control
 
 .Procedure
 
-. Create a file, such as `addresspool.yaml`, with content like the following example:
+. Create a file, such as `ipaddresspool.yaml`, with content like the following example:
 +
 [source,yaml]
 ----
-apiVersion: metallb.io/v1alpha1
-kind: AddressPool
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
 metadata:
   namespace: metallb-system
   name: doc-example
```

Comment:
Include label in YAML as it is relevant for advertisements.
e.g. in metadata
  labels:
    zone: east